code-challenges.test.js

```diff
@@ -65,5 +163,58 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe ("vowelIndex", () => {
+  it ("takes in a string and logs the index of the first vowel.", () => {
+    const vowelTester1 = "learn"
+    const vowelTester2 = "academy"
+    const vowelTester3 = "challenges"
+    expect (vowelIndex(vowelTester1)).toEqual(1)
+    expect (vowelIndex(vowelTester2)).toEqual(0)
+    expect (vowelIndex(vowelTester3)).toEqual(2)
+  })
+})
+// Successful Failure
+// ● vowelIndex › takes in a string and logs the index of the first vowel.
+//
+//   ReferenceError: vowelIndex is not defined
+
 
 // b) Create the function that makes the test pass.
+// Pseudo code
+// Create a function that takes in a string and logs the index of the first vowel
+// Input is a string- need to use an array to determine indexof --> create new variable with just vowels in an array called vowels
+// Will need function to iterate each letter in the string and determine whether it is a vowel or not-> for loop? and .length
+// Will need an if statement to compare index value
+// return index of first vowel of each word
+// Expected output : 1 , 0, 2
+// Note for Instructor: This challenge question took me over an hour to figure out, and I am still a bit confused on the function portion below. I had several classmates help me work through the remainder of the challenge once I got stuck after my if statement, around line 199 or so. I also tried using all of my resources through google but for some reason it was really difficult to find concrete instructions on how to solve this problem.
+
+
+const vowels = ['a', 'e', 'i', 'o', 'u'];
+console.log(vowels);
+
+const vowelIndex = (value) => {
+  let foundIndexes = []
+  for(let i = 0; i < value.length ; i++) {
+    if (vowels.indexOf(value[i]) > -1){
+      foundIndexes.push(i);
+    }
+  }
+  return foundIndexes.sort()[0]
+}
+console.log(vowelIndex(vowelTester1))
+console.log(vowelIndex(vowelTester2))
+console.log(vowelIndex(vowelTester3))
+
```

Comment:
Your logic looks great. I think the sort may be superfluous as the first vowel should be the first item in the array.